specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerActiveSecurityUserRulesList.json

```diff
@@ -0,0 +1,81 @@
+{
+  "title": "Get Azure Virtual Network Manager Active Configuration",
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subscriptionA",
+    "resourceGroupName": "myResourceGroup",
+    "networkManagerName": "testNetworkManager",
+    "parameters": {
+      "regions": [
+        "westus"
+      ],
+      "$skipToken": "fakeSkipTokenCode"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "region": "westus",
+            "commitTime": "2020-07-10T18:03:22.2578238+05:30",
+            "id": "/subscriptions/subscriptionA/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/securityUserConfigurations/myTestSecurityConfig/ruleCollections/testRuleCollection/rules/SampleUserRule",
```

Comment:
>SampleUserRule [](http://example.com/codeflow?start=231&length=14)

SampleSecurityUserRule?

Reply:
It is just an example name, any string would work